src/app/helptext/sharing/afp/afp.ts

```diff
@@ -2,6 +2,19 @@ import { Validators } from "@angular/forms";
 import { T } from "app/translate-marker";
 
 export const helptext_sharing_afp = {
+  fieldset_general: T('General Options'),
+  fieldset_permissions: ('Permissions'),
+  fieldset_allow: T('Allow'),
+  fieldset_deny: T('Deny'),
+  fieldset_ro: T('Read Only'),
+  fieldset_rw: T('Read/Write'),
+  fieldset_hostsallow: T('Hosts Allow'),
```

Comment:
I would flip these around to "Allow Hosts" and "Deny Hosts" to make more sense to the user.